docs/v0.1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Home"
+layout: default
+permalink: /
+redirect_from:
```

Comment:
This will break

Reply:
So what should it be?

Reply:
In older versions you don't want to have this because if you'll go to docs/v0.1/ you'll be redirected to /.

But best to test locally before removing

Reply:
Yes, you get redirected back to the main page. Let's see.

Reply:
Hmm, it's gone, but it is no redirected to a unknown page. 🤔